mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/entity/VerifiedClaim.java

```diff
@@ -0,0 +1,37 @@
+package io.mosip.esignet.mock.identitysystem.entity;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Data
+@Entity
+public class VerifiedClaim {
+
+    @Id
+    @Column(name = "id")
+    private String id;
+
+    @Column(name = "individual_id")
+    private String individualId;
+
+    private String claim;
```

Comment:
Why is there no @column annotation on claim field?

Reply:
added